src/view/container/creation_page.rs

```diff
@@ -713,7 +754,16 @@ impl CreationPage {
         let create_opts = if cmd.is_empty() {
             create_opts
         } else {
-            create_opts.command([cmd.as_str()])
+            let args = imp
+                .cmd_args
+                .borrow()
+                .to_owned()
+                .to_typed_list_model::<model::CmdArg>()
+                .into_iter()
+                .map(|arg| arg.arg_take());
+            let mut cmd = vec![cmd.to_string()];
+            cmd.extend(args);
+            create_opts.command(&cmd)
```

Comment:
Thanks, @vv9k, for working on this. :-)

> This work is mainly based on implementation of env-vars as I'm still trying to wrap my head around the code base so it might not be the best solution.

I hope it's not because of my programming skills. :-P But yes, all the glib2 and GTK specific stuff in the Rust bindings do take some time getting used to.

I would make the suggestion to solve the problem in a simpler way by just splitting across the whitespaces.

```suggestion
            create_opts.command(cmd.split_whitespace().collect::<Vec<_>>())
```

What do you think?

Reply:
> I hope it's not because of my programming skills. :-P But yes, all the glib2 and GTK specific stuff in the Rust bindings do take some time getting used to.

No :laughing:, on the contrary I think your skills made it easier for me to get accustomed to the glib/GTK ways.

> I would make the suggestion to solve the problem in a simpler way by just splitting across the whitespaces.
> 
> What do you think?

It will only work for simple cases, it will break once you try to pass in a path containing whitespace in it or a multi word string enclosed in `"`

For example:
 * I create a file in the container that has whitespace in it like /root/my\ cool\ file` and I try to reference it from pods when creating a container I would get 3 separate arguments like `["myprogram", "/root/my\", "cool\", "file"]` which would be interpreted differently than `["myprogram", "/root/my\ cool\ file"]`
  * Also trying to pass a string that contains spaces like `myprogram "some multiword string"` would end up as `["myprogram", "\"some", "multiword", "string\""]` instead of `["myprogram", "\"some multiword string \""]`

Reply:
Yeah, your points make sense. So, I think this is the way to go.

But, we should alter the overall design a bit to bring the "command" row and the "argument" list closer together, as they are now spatially disconnected. I already had an idea.

Will get back to you this evening when I have more time.